package/boot/uboot-envtools/files/ath79

```diff
@@ -16,6 +16,8 @@ case "$board" in
 alfa-network,ap121f|\
 buffalo,bhr-4grv2|\
 engenius,ecb1750|\
+glinet,gl-ar300m-lite|\
```

Comment:
Is adding lite here really connected to the introduction of the NAND target or should this go into a separate commit?

Reply:
Considering that the introduction of the NAND target and the GL-AR300M series and GL-AR750S support are tied together and that the commits are already organized in terms of enabling the NAND target and then using it, that seems a moot question.
```
f79db282e5 ath79: GL-AR750S: Add I2C Support
3425619d5b ath79: GL-AR300M series: Add I2C Support
4c91a9d1cc ath79: Refactor ath9k/ath10k sh functions into ath79's /lib/functions
d972e9fe97 ath79: GL-AR750S: Provide NAND Support
33826ca2d9 ath79: Add GL.iNet GL-AR300M16 as NOR-only Board
91e8084d02 ath79: GL-AR300M: Provide NAND support

cc9a863f28 sysupgrade: NAND: Prefer CONTROL for board, improve robustness
e41bcd1e2e build: sysupgrade-tar alt-board= for legacy upgrades
28f2d0ab12 ath79: Prepare NAND subtarget for upstream support of SPI NAND
7dcbf0b51a kernel: mtd: spinand: Backport chip definitions
1d728d6ee6 ath79: Remove legacy GL.iNet GL-AR300M NAND target
```

Reply:
Well, can't help you with the motivation.
But this particalur line looks like the ar300m-lite was just missing in package/boot/uboot-envtools/files/ath79. If you say that's true, one could also make it a separate fix unrelated to this patchset.